### DISCUSSION TITLE
诶🤓👆豪堪😋

### DISCUSSION THREAD

**Discussion by cuijiaqi0406** [share121/inter-knot#5299] [灌水]
🤓
![Screenshot_20240922_145134_com bilibili studio_edit_4103562655152](https://github.com/user-attachments/assets/623a44a1-57e8-49a4-8520-38a0cd8c5155)

**Comment by cuijiaqi0406** [comment]
链接：https://b23.tv/rO2niLA<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by woyaowanmhy666** [comment]
好家伙，我还以为是never will give you up呢。

**Comment by RWHwaxili** [comment]
我一看就不是never gonna give you up
被骗的太多了连链接都记下来了
又多了一个要背的链接

**Reply by Hjhnull** [reply]
https://m.youtube.com/watch?v=dQw4w9WgXcQ&pp=ygUXbmV2ZXIgZ29ubmEgZ2l2ZSB5b3UgdXA=

**Reply by Hjhnull** [reply]
https://github.com/user-attachments/assets/b0d28ba6-cc7c-4342-a661-d64e4903cd83

**Reply by dzqwe** [reply]
怎么做到的？教教我。

**Comment by Shirin-k423** [comment]
一天上一当，当当不一样

**Comment by Hjhnull** [comment]
好康的:https://m.youtube.com/watch?v=dQw4w9WgXcQ&pp=ygUXbmV2ZXIgZ29ubmEgZ2l2ZSB5b3UgdXA=

**Comment by 210flight** [comment]
你这码里面怎么藏着个甘雨🤔
